examples/platforms/da15000/radio.c

```diff
@@ -34,13 +34,14 @@
 #include <openthread/openthread.h>
 #include <openthread/platform/alarm.h>
 #include <openthread/platform/radio.h>
-
+#include <string.h>
```

Comment:
Can this include be moved down into its own block?  String.h is a standard header, so it doesn't make sense to include it mixed in with OT headers.